### DISCUSSION TITLE
如果分享的配置每一段每一个字符都有详细的注释和解释就好了

### DISCUSSION THREAD

**Discussion by 57382** [IrineSistiana/mosdns#680] [Q&A]
比如这一段：
 动态域名跳过缓存
      - matches: "!qname 00006801.com"
不懂这里"!"和“00006801.com”在这里的作用是什么。
查了mosdns wiki关于“!”的解释：
    - "!qname $ad_domain"   # !前缀代表取反。
看了还是不懂。

我只是想在别人分享的配置之上做二次修改，为了符合我的需要，而不是复制粘贴
但是看不懂就没办法自己改

**Comment by djoldwang** [comment]
chatgpt 问一下不就出来了
